compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/BallerinaParser.java

```diff
@@ -160,6 +155,8 @@ public STNode resumeParsing(ParserRuleContext context, STNode... parsedNodes) {
                 return parseWhileKeyword();
             case BOOLEAN_LITERAL:
                 return parseBooleanLiteral();
+            case RETURN_KEYWORD:
+                return parseReturnStatement();
```

Comment:
Shouldn't this be `parseReturnKeyword()`?

Reply:
ahh yeah. thanks